concepticondata/conceptlists/Hartmann-2013-162.tsv

```diff
@@ -1,163 +1,163 @@
+Hartmann-2013-162-2	2	ASK (a question)				1309	ASK (INQUIRE)	She asked me a difficult question	A asks R Q	New meaning	ask-a-question
+Hartmann-2013-162-3	3	ASK (about)						The man asked the women about a place to rent.	X asks Y about Z	New meaning	ask-about
+Hartmann-2013-162-4	4	ASK FOR	prositʹ	bitten	ask	1828	ASK (REQUEST)	The boy asked his parents for money.	A asks (X) for Y	Core list	ask-for
+Hartmann-2013-162-5	5	ASSASSINATE						The sniper assassinated the politician.	A assassinates P	New meaning	assassinate
+Hartmann-2013-162-6	6	BE A DRESS						This piece of cloth is actually a dress.	X is a dress	New meaning	be-a-dress
+Hartmann-2013-162-7	7	BE A HUNTER		Jäger sein	be a hunter	1053	HUNTER	This man is a hunter.	S is a hunter	Core list	be-a-hunter
+Hartmann-2013-162-8	8	BE BRIGHT				679	BRIGHT	The light is bright.	S is bright	New meaning	be-bright
+Hartmann-2013-162-9	9	BE BURDENED						The man is burdened with a problem.	A is burdened with P	New meaning	be-burdened
+Hartmann-2013-162-10	10	BE COLD			be cold	1287	COLD	The water is cold.	X is cold	New meaning	be-cold
+Hartmann-2013-162-11	11	BE DRY		trocken sein	be dry	1398	DRY	The ground is dry.	S is dry	Core list	be-dry
+Hartmann-2013-162-12	12	BE FAR FROM THE TRUTH						His reasoning is far from the truth.	A is far from the truth	New meaning	be-far-from-the-truth
+Hartmann-2013-162-13	13	BE GLAD				2192	REJOICE	The woman is glad.	A is glad.	New meaning	be-glad
+Hartmann-2013-162-14	14	BE HUNGRY	bytʹ golodnym	Hunger haben	be hungry OR feel hungry	1533	BE HUNGRY	The baby is hungry.	E is hungry	Core list	be-hungry
+Hartmann-2013-162-15	15	BE ILL		krank sein	be sick OR feel sick	1847	SICK	The little boy is sick.	S is sick	Old meaning	be-ill
+Hartmann-2013-162-16	16	BE NAMED				180	CALL BY NAME	The man was named Peter.	S is named X	New meaning	be-named
+Hartmann-2013-162-17	17	BE NEEDED				2339	NEED (SOMETHING)	This is severely needed.	S is needed	New meaning	be-needed
+Hartmann-2013-162-18	18	BE SAD	grustitʹ	traurig sein OR trauern	be sad OR feel sad	699	SAD	The little girl was sad.	E is sad	Core list	be-sad
+Hartmann-2013-162-19	19	BE SAD ABOUT						The man is sad about his loss.	E is sad about S	New meaning	be-sad-about
+Hartmann-2013-162-20	20	BEAT	bitʹ	schlagen	beat	1665	BEAT	The boy beat the snake with a stick.	A beats P (with I)	Core list	beat
+Hartmann-2013-162-21	21	BLINK	morgnutʹ	blinzeln	blink	1655	BLINK	I blinked (my eyes).	S blinks	Core list	blink
+Hartmann-2013-162-22	22	BOIL	kipetʹ	kochen (2)	boil	792	BOIL (OF LIQUID)	The water is boiling.	S boils	Extended list	boil
+Hartmann-2013-162-23	23	BOIL (tr)		kochen (1)		2492	BOIL (SOMETHING)	The cook boiled the potatoes.	A boils P	New meaning	boil-tr
+Hartmann-2013-162-24	24	BREAK	slomatʹ	zerbrechen	break	680	BREAK (DESTROY OR GET DESTROYED)	The boy broke the window with a stone.	A breaks P (with I)	Core list	break
```

Comment:
I think given the context it would be possible to narrow down the concept to BREAK (CLEAVE). What do you think?